src/signals/incident-management/containers/IncidentSplitContainer/IncidentSplitForm.js

```diff
@@ -0,0 +1,101 @@
+import React, { useCallback } from 'react';
+import PropTypes from 'prop-types';
+import { useForm } from 'react-hook-form';
+import { useHistory } from 'react-router-dom';
+
+import { subcategoriesType } from 'shared/types';
+
+import CONFIGURATION from 'shared/services/configuration/configuration';
+
+import { Heading } from '@datapunt/asc-ui';
+
+import {
+  StyledButton,
+  StyledDefinitionList,
+  StyledForm,
+  StyledSubmitButton,
+  FormWrapper,
+} from './styled';
+
+import IncidentSplitFormIncident from './IncidentSplitFormIncident';
+
+const IncidentSplitForm = ({ parentIncident, subcategories, onSubmit }) => {
+  const { control, handleSubmit, register } = useForm();
+
+  const history = useHistory();
+
+  const onCancel = useCallback(() => {
+    history.push(CONFIGURATION.INCIDENTS_ENDPOINT);
+  }, [history]);
+
+  return (
+    <FormWrapper>
+      <StyledForm onSubmit={handleSubmit(data => onSubmit(data))} data-testid="incidentSplitForm">
+        <Heading>Deelmelding maken</Heading>
+
+        <fieldset>
+          <Heading forwardedAs="h2">Hoofdmelding</Heading>
+
+          <StyledDefinitionList>
+            <dt>Melding</dt>
+            <dd data-testid="incidentSplitFormParentIncidentId">{parentIncident.id}</dd>
```

Comment:
Can we use destructuring here?

```
const {id, statusDisplayName, ...} = parentIncident;
```

Reply:
Personally I would prefer to create a new component for definition list, r,

In this context I would prefer not to destructure, IMHO: `parentIncident.id` is more clear then `id` .